packages/storybook/stories/va-radio-uswds.stories.jsx

```diff
@@ -288,8 +288,8 @@ Default.args = {
 };
 Default.argTypes = propStructure(radioDocs);
 
-export const Tiled = USWDSTiled.bind(null);
-Tiled.args = {
+export const Tile = USWDSTiled.bind(null);
```

Comment:
Discussed with Natalie because she noticed that USWDS lables this display type as "Tile" and not "Tiled" so I have updated that for the Storybook example for va-radio and va-checkbox.